package.json

```diff
@@ -53,6 +53,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.3.4",
     "redux": "^3.0.4",
+    "redux-devtools": "^2.1.5",
```

Comment:
I can of course deep-equal check with initial state instead. Then we could remove devtools as a dep